src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1531,12 +1468,72 @@ paths:
                   $ref: '#/components/schemas/Group'
         default:
           $ref: '#/components/responses/defaultResponse'
+  /groups/{gid}/user/{uid}:
+    parameters:
+      - name: gid
+        in: path
+        required: true
+        description: Group id
+        schema:
+          type: string
+      - name: uid
```

Comment:
Please change parameter name to `userId` to be consistent.

Reply:
Done